### PR TITLE
#3187 - Content: Institution: Offerings Upload: Add bullets and links Fixes

### PR DESCRIPTION
**Pull request by andrepestana-aot**
Updated offerings upload:
image

### CODE REVIEW

File: sources/packages/web/src/views/institution/OfferingsUpload.vue [hunk 1]
```diff
@@ -12,15 +12,42 @@
...
+            href="https://studentaidbc.ca/sims-institution-access"
+            rel="noopener"
+            target="_blank"
+            >Student Information Management System (SIMS) Institution Account
+            access</a
+          >
+          page
         </li>
-        <li>Please save the file as "CSV UTF-8 (Comma delimited) (*.csv)"</li>
+        <li>Please save the file as "CSV (Comma delimited)(*.csv)"</li>
```
Comment (dheepak-aot) [inline sources/packages/web/src/views/institution/OfferingsUpload.vue:50]: Please remove the `"`at the end

**Comment by dheepak-aot** [comment]
Just a minor comment.

**Review by dheepak-aot — APPROVED** [review]
Looks good 👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrepestana-aot

**Pull request merged**